readthedocs/templates/projects/webhook_list.html

```diff
@@ -0,0 +1,56 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Webhooks" %}{% endblock %}
+
+{% block nav-dashboard %} class="active"{% endblock %}
+
+{% block project-webhooks-active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Webhooks" %}{% endblock %}
+
+{% block project_edit_content %}
+  <p class="help_text">Something about webhooks</p>
```

Comment:
Probably should add a real short description here :D

Reply:
This is missing the _real description_ :)